packages/docs/docs/subscriptions/subscription-extensions.md

```diff
@@ -64,7 +64,7 @@ Below are explanations of the different extensions Medplum Provides
 ## Interactions
 
 :::caution Note
-By default, FHIR Subscriptions will execute on "create", "update", "delete" operations.
+By default, FHIR Subscriptions will execute on "create" and "update" operations.
 :::
 
```

Comment:
I think part of the confusion in this doc is that "create" and "delete" are not clearly separated.

I recommend we add a 3-# subheader here for `### Subscription for "create"-only events`

And then another one below for `### Delete subscriptions`

One limitation of the current model is that it appears we only support "create" or "delete", but not both.  We may want to add support for wildcard "*" or a comma separated list.